test/functional/test_framework/test_framework.py

```diff
@@ -129,6 +129,9 @@ def __init__(self):
         # skipped. If list is truncated, wallet creation is skipped and keys
         # are not imported.
         self.wallet_names = None
+        # By default the wallet is not required. Set to true by skip_if_no_wallet().
+        # When False, we ignore wallet_names regardless of what it is.
+        self.requires_wallet = False
```

Comment:
it's part of bitcoin#20267
May be rename this commit like 'partial ... 20267'?

Reply:
intentionally omitted since it's a very small portion of the larger PR and it's more appropriate to annotate it, at least imo, when backporting descriptor wallet support (which is what a majority of the changes center around)